src/lib/components/Avatar/Avatar.tsx

```diff
@@ -62,39 +65,41 @@ const AvatarImage = styled.img`
   object-fit: cover;
 `;
 
+const AvatarWrapper = styled.div`
```

Comment:
Needed to wrap everything for the status indicator to work. Previously it was fine without it but now with the overflow:hidden the indicator needs to be absolutely positioned relatively to the wrapper instead of the StyledAvatar